presentation-timestamps.md

```diff
@@ -0,0 +1,42 @@
+# `PaintTimingMixin`: Presentation timestamps and their meanings
+
+Aligning different paint-related timestamps in all the relevant specs.
+
+## Overview
+When originally introduced, paint timing ("`first-paint`" and "`first-contentful-paint`") aspired to represent an important moment in terms of user experience - "The pixels on the screen" representing a certain state.
+However, this is tricky in terms of interopability - that moment is not always measured in the same way, and is not part of the flow of operations covered by web standards.
+Originally in the paint-timing spec, a different time is used instead as the `startTime` - the end of the "update the rendering" phase, where the document is done setting up the rendering and hands over rendering to the user agent.
+This is confusing and non-interoperable, as Chromium still reports the VSync-time, which sufficiently to corresponds to "pixels on the screen".
```

Comment:
There's an extra "to" here -- I think either "sufficiently corresponds to" or "corresponds sufficiently to" is intended